### DISCUSSION TITLE
why the RP2040-ZERO do not work with this software?

### DISCUSSION THREAD

**Discussion by fooz2001** [fhdm-dev/scoppy#150] [Q&A]
hi i bought a RP2040-Zero board for this oscilloscope project because it have lower price than pico board the board have same processor but when i uploaded the software and connect my phone the app shows conected with board but no signal show in screen 

edit: i found a firmware v8 he worked but it is too old anybody have v14-13-12 or older?

**Comment by kd7eir** [comment]


**Reply by fooz2001** [reply]
the board have same pins because it have same microcontroller

**Reply by Bitflops** [reply]
Configure the GPIOs in the software through:

Settings>Firmware settings>GPIOs>....signal generator> 2

CH1 ADC is already GPIO26

Wire from GPIO 2 to GPIO 26 and set the Channel 1 voltage ranges:

Settings>Firmware settings>Channels>Channel 1> Min. Voltage 0,00  / Max. Voltage 3,30


![RP2040-Zero_Scoppy](https://github.com/user-attachments/assets/c7b72dab-4a82-4139-8e43-bcb2cf2e63f7)


Board: RP2040 Zero from Waveshare
Firmware: latest firmware V18 for Pi Pico
